controllers/JobController.js

```diff
@@ -132,101 +111,56 @@ exports.JobsByEmail = [
  * @returns {Object}
  */
 exports.addJob = [
-	auth,
-	body("title", "Title must not be empty.").isLength({ min: 1 }),
-	body("description", "Description must not be empty.").isLength({ min: 1 }),
-	body("cost", "Cost must be more than $10.").isLength({ min: 1 }).trim(),
-	body("owner", "Owner must not be empty.").isLength({ min: 1 }),
-	body("email", "Email must not be empty.").isLength({ min: 1 }).trim(),
-	body("skills", "Skills must not be less than 3 character.").isLength({ min: 3 }).trim(),
-	sanitizeBody("*").escape(),
-	(req, res) => {
-		try {
-			const errors = validationResult(req);
-			if (!errors.isEmpty()) {
-				return apiResponse.validationErrorWithData(res, "Validation Error.", errors.array());
-			}
-			else {
-				var Job = new JobModel(
-					{
-						title: req.body.title,
-						description: req.body.description,
-						cost: req.body.cost,
-						owner: req.body.owner,
-						email: req.body.email,
-						skills: req.body.skills,
-						date: new Date().toLocaleString('en-US'),
-						status: 'Active',
-						applicants: []
-					});
-				//Save Job.
-				Job.save(function (err) {
-					if (err) { return apiResponse.ErrorResponse(res, err); }
-					let JobData = new jobData(Job);
-					return apiResponse.successResponseWithData(res, "Job add Success.", JobData);
-				});
-			}
-		} catch (err) {
-			//throw error in json response with status 500.
-			return apiResponse.ErrorResponse(res, err);
-		}
-	}
-];
+  auth,
+  body("title", "Title must not be empty.").notEmpty(),
+  body("description", "Description must not be empty.").isLength({ min: 1 }),
+  body("postedBy", "Owner must not be empty.").isLength({ min: 1 }),
+  body("skills", "Skills must not be less than 3 character.").notEmpty(),
+  body("locations", "Locations must not be empty.").notEmpty(),
+  body("vacancies", "Vacancies must not be empty.").notEmpty(),
+  (req, res) => {
+    try {
+      const user = req.user;
 
-/**
- * Job update.
- *
- * @param {string}      jobId
- * @param {string}      applicantId
- *
- * @returns {Object}
- */
-exports.JobApplied = [
-	auth,
-	body("jobId", "Job ID must not be empty.").isLength({ min: 1 }).trim(),
-	body("applicantId", "Applicant ID must not be empty.").isLength({ min: 1 }).trim(),
-	sanitizeBody("*").escape(),
-	(req, res) => {
-		try {
-			const errors = validationResult(req);
-			// var Job = new JobModel();
+      if (user.userType !== constants.userTypes.RECRUITER)
+        return apiResponse.unauthorizedResponse(res, "Invalid Permission.");
+
+      const errors = validationResult(req);
+      if (!errors.isEmpty()) {
+        return apiResponse.validationErrorWithData(
+          res,
+          "Validation Error.",
+          errors.array()
+        );
+      } else {
+        var Job = new JobModel({
+          title: req.body.title,
+          description: req.body.description,
+          skills: req.body.skills,
+          locations: req.body.locations,
+          vacancies: req.body.vacancies,
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 5 tabs but found 10 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
					vacancies: req.body.vacancies,
```